postepy-higieny-i-medycyny-doswiadczalnej.csl

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" initialize-with-hyphen="false" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl" default-locale="pl-PL">
+  <info>
+    <title>Postępy Higieny i Medycyny Doświadczalnej</title>
+    <title-short>PHiMD</title-short>
+    <id>http://www.zotero.org/styles/postepy-higieny-i-medycyny-doswiadczalnej</id>
+    <link rel="self" href="http://www.zotero.org/styles/postepy-higieny-i-medycyny-doswiadczalnej"/>
+    <link href="http://www.phmd.pl/page.php?IDpage=117" rel="documentation"/>
+    <author>
+      <name>Tomasz Hryszko</name>
+      <email>tomhry@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>1732-2693</issn>
+    <updated>2016-09-17T20:46:54+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="name">
+    <names variable="author" suffix=": ">
+      <name et-al-min="16" et-al-use-first="15" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
+    </names>
+  </macro>
+  <citation collapse="citation-number">
+    <layout delimiter="," prefix="[" suffix="]">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+  <bibliography>
+    <sort>
+      <key variable="author"/>
+    </sort>
+    <layout suffix=";">
+      <text variable="citation-number" prefix="[" suffix="] "/>
+      <choose>
+        <if type="article-journal">
+          <text macro="name"/>
+          <text variable="title" suffix=". "/>
+          <text variable="container-title" form="short" suffix=", "/>
+          <date date-parts="year" form="text" variable="issued" suffix="; "/>
+          <text variable="volume" suffix=": "/>
+          <text variable="page"/>
+        </if>
+        <else-if type="webpage" match="any">
+          <text variable="container-title" suffix=". "/>
+          <text variable="URL"/>
+          <date delimiter="" variable="accessed" prefix="(" suffix=")">
```

Comment:
This seems wrong - it'd attach the date right to the URL - presumably you want a space -- add a `suffix=" "` to the URL or a space before the parenthesis in the prefix here.